task03-one-piece-website/js/get-data.js

```diff
@@ -0,0 +1,74 @@
+"use strict";
+
+const URL = "https://static.akurey.com/trainings/OnePieceInformation.json";
+const IMG_DIR = "../assets/img/";
+
+function getData(url) {
+  return fetch(url).then(response => response.json()).catch(error => {
+    console.log("Error fetching data: " + error);
+  });
+}
+
+function replaceContent(content, element, property) {
+  let elements = document.querySelectorAll(element);
+  for (let index = 0; index < elements.length; ++index) {
+    elements[index][property] = content[index];
+  }
+}
+
+function updateSections() {
+  getData(URL).then(function(onePieceData) {
+    sortData(onePieceData);
+    updateImgPaths(onePieceData);
+    // character section
+    updateSection(onePieceData.characters, ["name", "img"], [".character__name", ".character__img"], ["innerHTML", "src"]);
+
+    // islands section
+    updateSection(onePieceData.islands, ["name", "img", "Location"], 
+                                        [".card-label__name--island", ".island__img", ".card-label__description--island>span"],
+                                        ["innerHTML", "src", "innerHTML"]);
+
+    // mystic objects section
+    updateSection(onePieceData.mysticObjects, ["name", "img", "description"],
+                                              [".card-label__name--mist", ".mist-object__img", ".card-label__description--mist"],
+                                              ["innerHTML", "src", "innerHTML"]);
```

Comment:
Would be better if the repeated arrays were stored into a constant here in the function, and just pass them through the function call. And so on with whatever you can group
ie:
`const properties = ["innerHTML", "src", "innerHTML"]`

And so on.